### PR TITLE
[SMALLFIX] Support unmounting in alluxio-mount.sh

### PR DESCRIPTION
**Pull request by aaudiber**
This adds `Umount` and `SudoUmount` alongside the existing `Mount` and `SudoMount`.

### CODE REVIEW

**Comment by aaudiber** [comment]
alluxio-bot, check this please

File: bin/alluxio-mount.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ if [[ "$-" == *x* ]]; then
 fi
 BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
 
-USAGE="Usage: alluxio-mount.sh [Mount|SudoMount] [MACHINE]
+USAGE="Usage: alluxio-mount.sh --umount [Mount|SudoMount|Umount|SudoUmount] [MACHINE]
```
Comment (LuQQiu) [inline bin/alluxio-mount.sh]: Why adding `--umount` here?
Would it be better to use `alluxio-mount.sh [Mount|SudoMount|Umount|SudoUmount] [MACHINE]` directly?

Comment (calvinjia) [inline bin/alluxio-mount.sh]: Do we need `--umount`? If we do, do we need to update alluxio-start?

File: bin/alluxio-mount.sh [hunk 2]
```diff
@@ -85,44 +75,100 @@ function mount_ramfs_linux() {
...
+    *)
+      echo -e ${USAGE} >&2
+      exit 1
+  esac
 }
 
 function main {
   case "$1" in
-    Mount|SudoMount)
+    Mount|SudoMount|Umount|SudoUmount)
```
Comment (calvinjia) [inline bin/alluxio-mount.sh]: The new options don't seem to be used?

File: bin/alluxio-start.sh [hunk 3]
```diff
@@ -35,9 +35,8 @@ Where ACTION is one of:
   restart_workers    \tRestart any failed workers on worker nodes.
 
 MOPT (Mount Option) is one of:
-  Mount    \tMount the configured RamFS. Notice: this will format the existing RamFS.
-  SudoMount\tMount the configured RamFS using sudo.
-           \tNotice: this will format the existing RamFS.
+  Mount    \tMount the configured RamFS if it is not already mounted.
+  SudoMount\tMount the configured RamFS using sudo if it is not already mounted.
```
Comment (calvinjia) [inline bin/alluxio-start.sh]: Is the description still accurate?

File: bin/alluxio-mount.sh (hunk 1 above)
Comment (aaudiber) [reply bin/alluxio-mount.sh]: good catch, I was originally thinking of using `--umount`, but then changed it. I will delete `--umount`

File: bin/alluxio-mount.sh (hunk 2 above)
Comment (aaudiber) [reply bin/alluxio-mount.sh]: they are used later in `run_local`

File: bin/alluxio-start.sh (hunk 3 above)
Comment (aaudiber) [reply bin/alluxio-start.sh]: I believe so, we recently changed the behavior to not re-mount if a ramfs is already there: https://alluxio.atlassian.net/browse/ALLUXIO-3257

File: bin/alluxio-mount.sh (hunk 1 above)
Comment (aaudiber) [reply bin/alluxio-mount.sh]: sorry, the `--umount` was added accidentally. Removed it now

File: bin/alluxio-start.sh (hunk 3 above)
Comment (aaudiber) [reply bin/alluxio-start.sh]: That change is why the ability to manually un-mount is much more important now

File: bin/alluxio-mount.sh (hunk 2 above)
Comment (calvinjia) [reply bin/alluxio-mount.sh]: We are expecting users to directly call `alluxio-mount`?

Comment (aaudiber) [reply bin/alluxio-mount.sh]: Yes, is that different from how you view `alluxio-mount`?

**Pull request merged**